static/scss/payment.scss

```diff
@@ -19,6 +19,26 @@
     padding-top: 15px;
   }
 
+  .tac {
+    color: rgba(255, 255, 255, .7);
```

Comment:
Can you this color into a variable and use it elsewhere within this file?

Reply:
Done @noisecapella, found existing variable